src/app.tsx

```diff
@@ -11,21 +11,26 @@ import { CatContextProvider } from './contexts';
 import { ErrorFallback } from './components';
 import { ThemeProvider } from '@rneui/themed';
 import appTheme from './app-theme';
+import { DatadogProvider } from '@datadog/mobile-react-native';
+import DatadogConfig from './services/datadog';
+import { DdLogs } from '@datadog/mobile-react-native';
 
 const App = () => {
   const ErrorComponent = useCallback(() => <ErrorFallback />, []);
   return (
     <ErrorBoundary
-      onError={(e, stack) => console.error(`App Error: ${e} ${stack}`)}
+      onError={(e, stack) => DdLogs.error(`App Error: ${e} ${stack}`)}
```

Comment:
IMO, it is not advisable to expose the actual logger to consumer of the application.

Like backend, we should create an abstractor 'Logger' which uses the right transport layer to do the actual logging.

This approach is tactically correctly, but not flexible from an architecture standpoint